infrastructure/cdn-in-a-box/traffic_ops_integration_test/Dockerfile

```diff
@@ -56,9 +57,11 @@ COPY ./go.mod ./go.sum /go/src/github.com/apache/trafficcontrol/
 WORKDIR /go/src/github.com/apache/trafficcontrol/traffic_ops/testing/api
 RUN go mod vendor -v
 
-RUN CGO_ENABLED=0 go test -c ./v1* -ldflags="-w -s" -o traffic_ops_v1_integration_test
-RUN CGO_ENABLED=0 go test -c ./v2* -ldflags="-w -s" -o traffic_ops_v2_integration_test
-RUN CGO_ENABLED=0 go test -c ./v3* -ldflags="-w -s" -o traffic_ops_v3_integration_test
+ENV CGO_ENABLED=0
+RUN go test -c ./v1 -ldflags="-w -s" -o traffic_ops_v1_integration_test
+RUN go test -c ./v2 -ldflags="-w -s" -o traffic_ops_v2_integration_test
+RUN go test -c ./v3 -ldflags="-w -s" -o traffic_ops_v3_integration_test
+RUN go test -c ./v4 -ldflags="-w -s" -o traffic_ops_v4_integration_test
```

Comment:
Can these be a single cache layer?

Reply:
Made it a single cache layer in 99c1b1af37